### DISCUSSION TITLE
Tischständer FDM

### DISCUSSION THREAD

**Discussion by Statusbit** [Blackymas/NSPanel_HA_Blueprint#273] [Show and tell]
Erstmal ein fettes Danke für die ganze Arbeit welche in das Projekt gesteckt wird.

Falls jemand einen passenden Tischständer sucht:
https://www.thingiverse.com/thing:5824946

Bei Sonderwünschen, auf dieser Basis gerne melden. Teil ist parametrisch aufgebaut und teilweise kann ich schnell Anpassungen (höher, breiter,steiler...) vornehmen.

**Comment by Blackymas** [comment]
sieht gut aus - kann man immer gebrauchen ;)
